src/menu_put.c

```diff
@@ -0,0 +1,42 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   menu_put.c                                         :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: gwolf <gwolf@student.42vienna.com>         +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/01/06 12:35:59 by gwolf             #+#    #+#             */
+/*   Updated: 2024/01/06 14:29:58 by gwolf            ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "menu_put.h"
+
+void	ft_menu_put_text(t_render *render)
+{
+	t_putinfo	put;
+
+	put.mlx_ptrs = render->mlx_ptrs;
+	put.pos = (t_vec2i){20, 20};
+	put.col = render->menu.font_col;
+	if (render->show_menu == false)
+		return (ft_put_str(put,
```

Comment:
Can that not be in one line?

Reply:
You're right, leftover after refactor. Changed!